sklearn/tests/test_random_projection.py

```diff
@@ -222,13 +224,15 @@ def test_random_projection_transformer_invalid_input():
 
 
 def test_try_to_transform_before_fit():
+    data, _ = make_sparse_random_data(sp.coo_matrix, n_samples, n_features, n_nonzeros)
```

Comment:
Here, this is is bit the same than for `get_feature_names_out`, we should probably parametrize even if not really needed.

Reply:
Because there will be a time when `sp.coo_matrix` won't be supported by all the dependencies in the CI tests, even though currently sparse matrix is still supported by all of them. Got it.